### ISSUE TITLE
Document all ambiguous links 

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2835]
Document ambiguous links

do before #2772 where we actually fix the link titles.

**Comment by Maroyafaied** [comment]
@cthomasONRR the sitemap that @AlexandracmONRR is working on should have external links documented so that might be a good place to start as well as the content inventory spreadsheet we were using when redesigning the site.

**Comment by Maroyafaied** [comment]
@cthomasONRR would you consider this as completed as part of the content audit? Feel free to close this issue if that's the case.

**Comment by cthomasONRR** [comment]
@Maroyafaied Yes! I will close this now.
